integration/tests/ipv6/ipv6_test.go

```diff
@@ -171,6 +191,29 @@ var _ = Describe("(Integration) [EKS IPv6 test]", func() {
 				}
 				return svcIP.Version()
 			}, 5*time.Second, time.Minute).Should(Equal(6))
+
+			By("ensuring workloads can run successfully")
+			test, err := kube.NewTest(params.KubeconfigPath)
+			Expect(err).ShouldNot(HaveOccurred())
+			d := test.CreateDeploymentFromFile(test.Namespace, "../../data/podinfo.yaml")
+			test.WaitForDeploymentReady(d, 10*time.Minute)
+
+			pods := test.ListPodsFromDeployment(d)
+			Expect(len(pods.Items)).To(Equal(2))
+
+			// For each pod of the Deployment, check we receive a sensible response to a
+			// GET request on /version.
+			for _, pod := range pods.Items {
+				Expect(pod.Namespace).To(Equal(test.Namespace))
+
+				req := test.PodProxyGet(&pod, "", "/version")
+				fmt.Fprintf(GinkgoWriter, "url = %#v", req.URL())
+
+				var js interface{}
+				test.PodProxyGetJSON(&pod, "", "/version", &js)
+
+				Expect(js.(map[string]interface{})).To(HaveKeyWithValue("version", "1.5.1"))
```

Comment:
hm how come we're hardcoding `1.5.1`?

Reply:
It's the version coming from `data/podinfo.yaml`:

```
      - name: podinfod
        image: stefanprodan/podinfo:1.5.1@sha256:702633d438950f3675d0763a4ca6cfcf21a4d065cd7f470446c67607b1a26750
```

Reply:
ah yep, thanks! :)